app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    operation, old_file_loc, new_file_loc = command.split()
+
+    if operation == "mv" and old_file_loc != new_file_loc:
+        new_path, _ = os.path.split(new_file_loc)
+        if new_path:
+            os.makedirs(new_path, exist_ok=True)
+
+        with open(old_file_loc, "r") as reader:
+            with open(new_file_loc, "w") as writer:
```

Comment:
no need to use "with" for 2 times
![image](https://user-images.githubusercontent.com/103201010/220621186-9ec96757-dfc8-4f00-935e-f5b160b57c7e.png)